wagtail/tests/test_audit_log.py

```diff
@@ -278,28 +278,31 @@ def test_page_delete(self):
                 title="Another child", slug="child-page-2", content="hello"
             )
         )
-
-        child.delete()
-        self.assertEqual(
-            PageLogEntry.objects.filter(action="wagtail.delete").count(), 1
+        child.add_child(
+            instance=SimplePage(
+                title="Grandchild", slug="grandchild-page", content="hello"
+            )
         )
 
-        # check deleting a parent page logs child deletion
-        self.home_page.delete()
+        # check deleting a parent page logs descendent deletion
+        with self.assertNumQueries(189):
```

Comment:
oops, missed this when merging...

We should avoid these sorts of 'magic numbers' in tests, as they're likely to be highly sensitive to unrelated changes to the logic. Will revert...